### PR TITLE
netstandard 1.3 support removed

### PR DESCRIPTION
**Pull request by klementi**
*Description of changes:*
netstandard 1.3 support removed (deprecated)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: Libraries/src/Amazon.Lambda.CloudWatchLogsEvents/Amazon.Lambda.CloudWatchLogsEvents.csproj [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <Description>Amazon Lambda .NET Core support - CloudWatchLogsEvents package.</Description>
     <TargetFrameworks>netstandard2.0;netcoreapp3.1</TargetFrameworks>
     <AssemblyTitle>Amazon.Lambda.CloudWatchLogsEvents</AssemblyTitle>
-    <VersionPrefix>2.1.0</VersionPrefix>
+    <VersionPrefix>3.0.0</VersionPrefix>
```
Comment (ashovlin) [inline Libraries/src/Amazon.Lambda.CloudWatchLogsEvents/Amazon.Lambda.CloudWatchLogsEvents.csproj:7]: I may have given conflicting advice originally, but after discussing with @normj  last week can we only do a minor version bump if the package didn't previously support .NET Standard 1.3?

File: Libraries/src/Amazon.Lambda.S3Events/Amazon.Lambda.S3Events.csproj [hunk 2]
```diff
@@ -4,9 +4,9 @@
```
Comment (ashovlin) [inline Libraries/src/Amazon.Lambda.S3Events/Amazon.Lambda.S3Events.csproj:4]: Can you take the `AWSSDK.S3` reference to 3.7 here too?

File: Libraries/src/Amazon.Lambda.CloudWatchLogsEvents/Amazon.Lambda.CloudWatchLogsEvents.csproj (hunk 1 above)
Comment (klementi) [reply Libraries/src/Amazon.Lambda.CloudWatchLogsEvents/Amazon.Lambda.CloudWatchLogsEvents.csproj]: Fixed

File: Libraries/src/Amazon.Lambda.S3Events/Amazon.Lambda.S3Events.csproj (hunk 2 above)
Comment (klementi) [reply Libraries/src/Amazon.Lambda.S3Events/Amazon.Lambda.S3Events.csproj:4]: Fixed

File: Libraries/test/TestFunction/TestFunction.csproj [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
     <AssemblyName>TestFunction</AssemblyName>
     <OutputType>Library</OutputType>
     <PackageId>TestFunction</PackageId>
-    <RuntimeFrameworkVersion>1.0.4</RuntimeFrameworkVersion>
-    <PackageTargetFallback>$(PackageTargetFallback);dnxcore50</PackageTargetFallback>
+    <!--<RuntimeFrameworkVersion>1.0.4</RuntimeFrameworkVersion>-->
```
Comment (normj) [inline Libraries/test/TestFunction/TestFunction.csproj:8]: Delete these commented out elements.

**Review by normj — APPROVED** [review]

**Review by ashovlin — APPROVED** [review]

**Pull request merged**